README.md

```diff
@@ -126,7 +126,7 @@ If you like this project, You can support it by giving it a star ⭐
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank My friends for the support and inspiration. 
 
```

Comment:
As requested by the previous code reviewer please consider removing the commented line as it has nothing to do with the user who is reading your README else it was there just to guide you when writing a professional README.

![image](https://user-images.githubusercontent.com/94223367/235122112-f420bb02-bbba-49c8-b8ea-25311178071c.png)


I am talking about the following text:-

`Give credit to everyone who inspired your codebase.`